Request: Add TypeScript types as dependencies instead of devDependencies

TL;DR: `@types/debug` is a `devDependency`. It should  be a `dependency`.

That's because a logger with its types is exported from `@node-wot/core`.

## Long story

I'm using the package `@node-wot/core` in a TypeScript project.

```
$ tsc -p .
node_modules/@node-wot/core/dist/logger.d.ts:1:19 - error TS7016: Could not find a declaration file for module 'debug'. 'wot-adapter/node_modules/@node-wot/core/node_modules/debug/src/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/debug` if it exists or add a new declaration (.d.ts) file containing `declare module 'debug';`

1 import debug from "debug";
                    ~~~~~~~
```

The package exports a logger function that returns a type of its dependency `debug`.
`@types/debug` is only listed as a devDependency.

As a workaround, I have to install `@types/debug` in my project that doesn't have `debug` as a direct dependency.

--- a/packages/binding-http/package.json
+++ b/packages/binding-http/package.json
@@ -24,7 +24,6 @@
         "@types/basic-auth": "1.1.3",
         "@types/eventsource": "^1.1.15",
         "@types/express": "^4.17.3",
-        "@types/node-fetch": "^2.6.12",
         "body-parser": "^1.20.3",
         "express": "^4.20.0",
         "ssestream": "^1.1.0",
@@ -32,6 +31,7 @@
     },
     "dependencies": {
         "@node-wot/core": "1.0.0",
+        "@types/node-fetch": "^2.6.12",
         "accept-language-parser": "1.5.0",
         "basic-auth": "2.0.1",
         "client-oauth2": "^4.2.5",

--- a/packages/core/package.json
+++ b/packages/core/package.json
@@ -16,13 +16,13 @@
     "types": "dist/core.d.ts",
     "devDependencies": {
         "@types/content-type": "^1.1.8",
-        "@types/debug": "^4.1.7",
         "@types/uritemplate": "^0.3.4",
         "@types/uuid": "^8.3.1"
     },
     "dependencies": {
         "@petamoriken/float16": "^3.1.1",
         "@thingweb/thing-model": "^1.0.4",
+        "@types/debug": "^4.1.7",
         "ajv": "^8.11.0",
         "ajv-formats": "^2.1.1",
         "cbor": "^8.1.0",

